plugins/auth-backend-module-okta-provider/src/module.test.ts

```diff
@@ -68,7 +66,7 @@ describe('authModuleOktaProvider', () => {
     expect(startUrl.pathname).toBe('/oauth2/v1/authorize');
     expect(Object.fromEntries(startUrl.searchParams)).toEqual({
       response_type: 'code',
-      scope: combinedScopes,
+      scope: additionalScopes,
```

Comment:
The behavior change here just highlights that passing `scope` to the strategy is broken and currently breaks in the same way if any scopes requested from the client. Followup fix will reintroduce the default scopes that we lose if we configure additional scopes here.